packages/moonstone/Dialog/Dialog.module.less

```diff
@@ -18,21 +24,15 @@
 	}
 
 	&.reserveClose {
-		.padding-start-end(@moon-dialog-h-padding, @moon-dialog-reserve-close-right-padding);
-
-		.moon-custom-text({
-			.padding-start-end(@moon-dialog-h-padding, @moon-dialog-reserve-close-right-padding-size-large);
-		});
+		.padding-start-end(@dialog-padding-left, @moon-dialog-padding-right); // Note the subtile name difference between these two variables. This is intentional.
 	}
 
-	.closeButton {
-		top: @moon-dialog-close-button-top;
-		right: @moon-dialog-close-button-right;
+	.closeContainer {
+		margin-top: @moon-popup-body-margin-top;
+		.margin-start-end(@moon-dialog-closecontainer-separation, auto);
 	}
 
 	.title {
-		.padding-start-end(0, @moon-dialog-title-right-padding);
```

Comment:
Should we add padding here for spacing between the title and the buttons?
<img width="593" alt="spacing" src="https://user-images.githubusercontent.com/8940009/62168647-db196500-b2da-11e9-936f-8ed04d09b00c.png">

Reply:
That's a good point. Think it should be the same as the Panels.Header title vs headerComponents spacing?

Wanna tackle that while I'm looking into and wrapping up the notification thing?

Reply:
Sure. I'll go with the `Header` specs